docs/move.md

```diff
@@ -130,11 +133,35 @@ This can now be achieved with the following procedure:
 
 9. Delete the bootstrap cluster
 
-**Important Note** The move process should be done when the BMHs are
-in a steady state. BMHs should not be moved while any operation is on-going i.e.
-BMH is in provisioning state. This will result in failure since the IP of the
-BMH might change after the move and the DHCP-leases from the management cluster
-are not moved to target cluster.
+## Impotant Notes
+
+The following requirements are essential for the move process to run
+successfully:
+
+1.
+The move process should be done when the BMHs are in a steady state. BMHs
+should not be moved while any operation is on-going i.e. BMH is in provisioning
+state. This will result in failure since the IP of the BMH might change after
+the move and the DHCP-leases from the management cluster are not moved to target
+cluster.
+
+2.
+Before the move process is initialized, it is important to delete the ironic
+pod/ironic containers. If ironic is deployed in cluster the deployment is named
+`metal3-ironic`, if it is deployed locally outside the cluster, the containers
+to be deleted are `ironic`, `ironic-inspector`, `ironic-endpoint-keepalived`,
+`ironic-dnsmasq`, `ironic-httpd` and `mariadb`. Deleting the ironic before move
+process makes sure that the old ironic does not interfere with the operations of
+the new ironic deployed in target cluster and also it makes sure that there is
+only one dnsmasq existent at any time.
+
+3.
+The provisioning bridge where the `ironic-endpoint-IP` is supposed to be
+attached should have a static IP assignment on it before the ironic
+pod/containers start to operate in the target cluster. This is important since
+`ironic-endpoint-keepalived` container will only assign the `ironic-endpoint-IP`
+on the provisioning bridge in target cluster when it has an IP on it. Otherwise
+it will fail to attach the IP and ironic will be unreachable.
```

Comment:
maybe specify that this interface is used to host the DHCP server so it cannot be configured to use DHCP ?

Reply:
Done. Thanks